server/uno.go

```diff
@@ -103,89 +95,60 @@ func contains(arr []string, val string) (int, bool) {
 ////////////////////////////////////////////////////////////
 // These are all of the functions for the game -> essentially public functions
 ////////////////////////////////////////////////////////////
-func updateGame(c echo.Context) *Response {
+func updateGame(game string, username string) bool {
 	success := false
-	if success = checkID(c.Param("game")); success && gameStarted {
-		return &Response{true, newPayload(c.Param("username"))}
+	if success = checkID(game); success && gameStarted {
+		return true
 	}
-	return &Response{false, nil}
+	return false
 }
 
-func createNewGame(c echo.Context) *Response {
-
-	ctx := context.Background()
-	client := createClient(ctx)
-
-	_, _, err := client.Collection("users").Add(ctx, map[string]interface{}{
-		"first": "Ada",
-		"last":  "Lovelace",
-		"born":  1815,
-	})
-	if err != nil {
-		log.Fatalf("Failed adding alovelace: %v", err)
-	}
```

Comment:
Is this to do with the database backend and that's why it's being removed?
If not, this should probably be part of the `routeHandlers.go` file.

Reply:
Yeah I watched him remove this in class -- this was an experiment with the Firebase emulator I think.